CONTRIBUTING.md

```diff
@@ -95,6 +95,11 @@ npm test #run all tests
 npm test -- --watch #re-run tests on file change
 ```
 
+### Testing the Daemon run logic
+The deamon tests in the `cli/daemon/run` use many auto generated files from the `e2e-tests/testdata` directory. To generate the
```

Comment:
Is this true? I cannot find any reference to `e2e-tests` in the `cli/` folder or any subdirectory.